package.json

```diff
@@ -8,11 +8,10 @@
     "url": "git://github.com/menpo/landmarker.git"
   },
   "scripts": {
-    "build": "DISABLE_NOTIFIER=true NODE_ENV=production gulp build",
-    "watch": "gulp",
+    "build": "NODE_ENV=production ./node_modules/gulp/bin/gulp.js build",
+    "watch": "./node_modules/gulp/bin/gulp.js",
```

Comment:
@lirsacc is it better here to use `./node_modules/gulp/bin/gulp.js` in leiu of `gulp` so users don't need to have installed with `-g`?